docker/Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM php:7.0-fpm-alpine
+
+ENV VARNISH_VERSION 5.0.0
```

Comment:
shouldn't this use a build arg instead, as it is an env var affecting the image building, not the runtime (it is useless at runtime AFAIK.

btw, if a build arg is used, it may be possible to overwrite it when building the image, so testing different versions in different Travis jobs. Using ``ENV``, it cannot be achieved (except by using ``sed`` to rewrite the Dockerfile)